packages/manager/modules/hub/src/e2eTests/hubHome/CA_EU_US_hub.e2e.js

```diff
@@ -0,0 +1,54 @@
+import config from '../../../../../tools/testcafe/config';
```

Comment:
```
import config from '@ovh-ux/testcafe-manager/config'
```
same for other imports

Reply:
it's working but i'm facing an issue with eslint:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md